features/step_definitions/basics_steps.rb

```diff
@@ -0,0 +1,20 @@
+When("I visit the {string}") do |string|
+  visit root_path
+end
+
+Then("I should see {string}") do |text|
+  expect(page).to have_content text
+end
+
+Then("I should see a navigation bar") do
+  find('.nav_bar').visible?
+end
+
+Then("I should see {string} link") do |string|
+  page.should have_link(string)
```

Comment:
Does it make more sense if we instead of using |string| as variables for links, maybe we should use |link| to make it easier for everyone to understand our step definitions?

Reply:
Changed!